src/main/archetype/ui.tests/test-module/specs/aem/forms.js

```diff
@@ -220,4 +221,21 @@ describe('AEM Forms OOTB Content Tests', () => {
 
     });
 
+    describe('Theme testing', () => {
```

Comment:
Are we removing the test case, if the `forms` addon is not enabled in the archetype ?

Reply:
@pankaj-parashar  can you comment on this?

Reply:
@rismehta , that has been done already when we added test for templates and fdm.
cc : @suryansh29